activity.js

```diff
@@ -26,33 +25,50 @@ class Activity {
   };
 
   countdown() {
-    startTimer.removeEventListener('click', starter);
     var that = this;
```

Comment:
Any reason for this = that besides style? It is a bit confusing to an outsider

Reply:
Inside of the iteration function 'this' doesn't seem to work as it usually does, so had to change it to 'that' to access attributes/methods. 

Definitely frustrating to figure out!